ci/archive.sh

```diff
@@ -8,4 +8,4 @@ source $root/ci/vars.sh
 
 cp nginx.conf dist
 
-tar -czf $APP.$EXT -C $root dist
+zip $APP.$EXT $root dist
```

Comment:
The `mvn dependency:get` command has `-Dpackaging=zip` set, and I cannot find documentation for what the string is for setting it to tgz (and omitting it defaults to jar, which is not right).